sample/src/debug/java/com/polidea/cockpit/sample/sampleparams/SamplePresenter.kt

```diff
@@ -36,12 +40,17 @@ class SamplePresenter(override val sampleView: SampleContract.View)
     override fun stop() {
         removeOnChangeListeners()
         removeCallbacks()
+        removeOnValueSelectedListeners()
     }
 
     private fun removeCallbacks() {
         Cockpit.removeVersionActionRequestCallback(versionActionRequestCallback)
     }
 
+    private fun removeOnValueSelectedListeners() {
+        Cockpit.removeOnFontSizeChangeListener(onFontSizeChangeListener)
```

Comment:
It should be `Cockpit.removeFontListSelectionChangeListener(fontSelectedListener)` here

Reply:
Oops!